
# Curated list of papers on variation within SARS-CoV2



(last update Sept 14, 2020)

https://www.ncbi.nlm.nih.gov/pubmed/32752979

Andrés C, Garcia-Cehic D, Gregori J, Piñana M, Rodriguez-Frias F, Guerrero-Murillo M, Esperalba J, Rando A, Goterris L, Codina MG, Quer S, Martín MC, Campins M, Ferrer R, Almirante B, Esteban JI, Pumarola T, Antón A, Quer J. Naturally occurring SARS-CoV-2 gene deletions close to the spike S1/S2 cleavage site in the viral quasispecies of COVID19 patients. Emerg Microbes Infect. 2020 Dec;9(1):1900–1911. PMID: 32752979

This study identified large number of deletion in S gene

Sequencing of amplified S gene fragments using Illumina (miSeq) PRJNA630679

Tags: Deletions, Amplicon, Illumina, S gene


-----

http://dx.doi.org/10.1080/22221751.2020.1782271

Gong Y-N, Tsao K-C, Hsiao M-J, Huang C-G, Huang P-N, Huang P-W, Lee K-M, Liu Y-C, Yang S-L, Kuo R-L, Chen K-F, Liu Y-C, Huang S-Y, Huang H-I, Liu M-T, Yang J-R, Chiu C-H, Yang C-T, Chen G-W, Shih S-R. SARS-CoV-2 genomic surveillance in Taiwan revealed novel ORF8-deletion mutant and clade possibly associated with infections in Middle East. Emerg Microbes Infect. 2020 Dec;9(1):1457–1466. PMID: 32543353

This study identified deletion of ORF8 in one Taiwanese sample. Other mutations identified are ORF1ab-V378I (novel), ORF8-L84S, ORF3a-G251V, and S-D614G.

WGS using Illumina, GISAID, no Read data

Tags: Deletion, ORF1ab-V378I, ORF8-L84S, ORF3a-G251V, and S-D614G, WGS

------

http://dx.doi.org/10.1038/d41586-020-02544-6

Callaway E. The coronavirus is mutating — does it matter? Nature. Nature Publishing Group; 2020 Sep 8;585(7824):174–177.

An opinion piece of the merits of the D614G and its implications

Tags: Opinion, Review

-------

http://dx.doi.org/10.1016/j.cell.2020.07.012

Li Q, Wu J, Nie J, Zhang L, Hao H, Liu S, Zhao C, Zhang Q, Liu H, Nie L, Qin H, Wang M, Lu Q, Li X, Sun Q, Liu J, Zhang L, Li X, Huang W, Wang Y. The Impact of Mutations in SARS-CoV-2 Spike on Viral Infectivity and Antigenicity. Cell. 2020 Sep 3;182(5):1284–1294.e9. PMCID: PMC7366990

![](https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/689a6d1a-07f1-44dc-9fb3-3644ad204a42/gr3.jpg)

Infectivity analysis of 106 S gene mutants (substitutions and glycosilation alterations) identified from GISAID sequences (May 6, 2020). The dominant D614G itself and combined with other
mutations are more infectious. Ablation of both N331 and N343 glycosylation at RBD
drastically reduced infectivity. Ten mutations such as N234Q, L452R, A475V, and V483A
was markedly resistant to some mAbs

Tags: Mutations effect analysis, S gene

--------

http://dx.doi.org/10.1016/j.cell.2020.08.012

Starr TN, Greaney AJ, Hilton SK, Ellis D, Crawford KHD, Dingens AS, Navarro MJ, Bowen JE, Tortorici MA, Walls AC, King NP, Veesler D, Bloom JD. Deep Mutational Scanning of SARS-CoV-2 Receptor Binding Domain Reveals Constraints on Folding and ACE2 Binding. Cell. 2020 Sep 3;182(5):1295–1310.e20. PMCID: PMC7418704

![](https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/01b3c8e1-9343-4e0a-9f40-887cdd50f33e/gr3.jpg)

Deep mutation scanning of all possible S gene (receptor binding domain) mutations and their affect on (1) expression and (2) affinity towards ACE2. This gives detailed function info about S protein amino acids 331 - 531.

Tags: Mutation effect analysis, S gene, RBD

-------

http://dx.doi.org/10.1101/gr.266221.120

Gomez-Carballa A, Bello X, Pardo-Seco J, Martinon-Torres F, Salas A. Mapping genome variation of SARS-CoV-2 worldwide highlights the impact of COVID-19 super-spreaders. Genome Res [Internet]. 2020 Sep 2; Available from: http://dx.doi.org/10.1101/gr.266221.120 PMID: 32878977

Analysis of GISAID data to identify temporal progression of SARS-VCov2 clades. Identifies 12 Nov 2019 as the TMCA for SARS-CoV2.

Tags: Clades, TMCA, Genomic diversity

-------

http://dx.doi.org/10.1016/j.meegid.2020.104525

Pereira F. Evolutionary dynamics of the SARS-CoV-2 ORF8 accessory gene. Infect Genet Evol. 2020 Sep 2;85:104525. PMCID: PMC7467077

"*These findings suggest that SARS-CoV-2 can persist without a functional ORF8 protein. Deletion breakpoints were found located in predicted hairpins suggesti g a possible involvement of these elements in the rearrangement process. Although the function of ORF8 remains to be elucidated, its structural plasticity and high diversity suggest an important role in SARS-CoV-2 pathogenicity.*"


Tags: Orf8

-----

http://dx.doi.org/10.1016/j.meegid.2020.104351

van Dorp L, Acman M, Richard D, Shaw LP, Ford CE, Ormond L, Owen CJ, Pang J, Tan CCS, Boshier FAT, Ortiz AT, Balloux F. Emergence of genomic diversity and recurrent mutations in SARS-CoV-2. Infect Genet Evol. 2020 Sep;83:104351. PMCID: PMC7199730

Analyzed GISAID data (April 19, 2020) and user SRA data to confirm some of the variable sites. This is a comprehensive analysis of variation progression from GISAID data. Cites SKP observable notebook.

Tags: Clades, TMCA, Genomic diversity

------

http://dx.doi.org/10.1016/j.ygeno.2020.04.016

Yin C. Genotyping coronavirus SARS-CoV-2: methods and implications. Genomics. 2020 Sep;112(5):3588–3596. PMCID: PMC7184998

A somewhat simplistic way to catalog SNP variation across early set of GISAID data.

Tags: Clades, TMCA, Genomic diversity

------

http://dx.doi.org/10.1073/pnas.2008281117

Dearlove B, Lewitus E, Bai H, Li Y, Reeves DB, Joyce MG, Scott PT, Amare MF, Vasan S, Michael NL, Modjarrad K, Rolland M. A SARS-CoV-2 vaccine candidate would likely match all currently circulating variants. Proc Natl Acad Sci U S A [Internet]. 2020 Aug 31; Available from: http://dx.doi.org/10.1073/pnas.2008281117 PMID: 32868447

Analysis of GISAID genomes (May 18, 2020) to understand whether global variation will affect vaccine efficacy. They did not find evidence that variation observed so far will create issues for vaccine design.

Tags: Genomic Diversity

-------

http://dx.doi.org/10.1101/2020.08.28.271965

Schlick T, Zhu Q, Jain S, Yan S. Structure-Altering Mutations of the SARS-CoV-2 Frame Shifting RNA Element. bioRxiv [Internet]. 2020 Aug 30; Available from: http://dx.doi.org/10.1101/2020.08.28.271965 PMCID: PMC7457599

The authors identify a minimum set of mutations that destroy programmed frame shifting required for orf1ab transcription: "*only two residue changes can accomplish a dramatic change in structure, namely destruction of Stem 2 and the pseudoknot*"

Tags: Fremeshifting

---------

http://dx.doi.org/10.1371/journal.pone.0238490

Li Y, Yang X, Wang N, Wang H, Yin B, Yang X, Jiang W. SNPs or RNA modifications? Concerns on mutation-based evolutionary studies of SARS-CoV-2. PLoS One. 2020 Aug 28;15(8):e0238490. PMCID: PMC7454988

Attempt (and a cautionary note) to highlight the fact that RNA modification can be responsible for some SNPs observed in SARS-CoV2 data.

Tags: RNA editing

-------

http://dx.doi.org/10.1016/j.virol.2020.08.010

Jones LR, Manrique JM. Quantitative phylogenomic evidence reveals a spatially structured SARS-CoV-2 diversity. Virology. 2020 Aug 26;550:70–77. PMCID: PMC7448746

Analysis of SARS-CoV2 global dispersion patterns based on GISAID (April 25, 2020) data.

Tags: Clades, Genomic diversity

-------

http://dx.doi.org/10.1101/2020.08.26.267831

Siqueira JD, Goes LR, Alves BM, de Carvalho PS, Cicala C, Arthos J, Viola JPB, de Melo AC, Soares MA. SARS-CoV-2 genomic and quasispecies analyses in cancer patients reveal relaxed intrahost virus evolution. bioRxiv [Internet]. 2020 Aug 26; Available from: http://dx.doi.org/10.1101/2020.08.26.267831 PMCID: PMC7457605

Analysis of intra-host SARS-CoV2 variation in cancer patients and non-cancer controls. Cancer patents have higher intra-host variability. 

Sequencing of ARTIC amplicons. Illumina sequencing. No data is provided.

Tags: Intrahost, Illumina, Amplicon, ARTIC, Nodata

------

http://dx.doi.org/10.1016/j.ijid.2020.08.052

Guan Q, Sadykov M, Mfarrej S, Hala S, Naeem R, Nugmanova R, Al-Omari A, Salih S, Mutair AA, Carr MJ, Hall WW, Arold ST, Pain A. A genetic barcode of SARS-CoV-2 for monitoring global distribution of different clades during the COVID-19 pandemic. Int J Infect Dis [Internet]. 2020 Aug 22; Available from: http://dx.doi.org/10.1016/j.ijid.2020.08.052 PMCID: PMC7443060

Development of a genotyping protocol that takes advantage of known SARS-CoV2 variants to quickly identify clades. 


Tags: Clades, Genomic diversity

------

http://dx.doi.org/10.1016/j.cell.2020.06.040 (relates to http://dx.doi.org/10.1016/j.cell.2020.06.043)

Grubaugh ND, Hanage WP, Rasmussen AL. Making Sense of Mutation: What D614G Means for the COVID-19 Pandemic Remains Unclear. Cell. 2020. p. 794–795. PMCID: PMC7332445

Authors are skeptical of the claim that D614G increases infection potential of SARS-CoV2: "*It is clear from the in vitro and clinical data that G614 has a distinct phenotype, but whether this is the result of bonafide adaptation to human ACE2, whether it increases transmissibility, or will have a notable effect is not clear. *"

Tags: S gene, D614G

-----

http://dx.doi.org/10.1038/s41598-020-70827-z

Isabel S, Graña-Miraglia L, Gutierrez JM, Bundalovic-Torma C, Groves HE, Isabel MR, Eshaghi A, Patel SN, Gubbay JB, Poutanen T, Guttman DS, Poutanen SM. Evolutionary and structural analyses of SARS-CoV-2 D614G spike protein mutation now documented worldwide. Sci Rep. 2020 Aug 20;10(1):14031. PMCID: PMC7441380

Analysis of D614G based on early GISAID (March 30, 2020) data and E-gene amplicon sequencing of local samples (no data is provided).

"*the global spread of the D614G mutation may have nothing to do with viral biology but may simply be a consequence of the high level of interconnectedness of Europe to the rest of the world. Tus, the emergence of the D614G clade may be explained by a founder event and subsequent clonal expansion in Europe that led to its spreading worldwide*".

Tags: S gene, D614G, Illumina, nodata

-----

http://dx.doi.org/10.1016/j.cell.2020.06.043

Korber B, Fischer WM, Gnanakaran S, Yoon H, Theiler J, Abfalterer W, Hengartner N, Giorgi EE, Bhattacharya T, Foley B, Hastie KM, Parker MD, Partridge DG, Evans CM, Freeman TM, de Silva TI, Sheffield COVID-19 Genomics Group, McDanal C, Perez LG, Tang H, Moon-Walker A, Whelan SP, LaBranche CC, Saphire EO, Montefiori DC. Tracking Changes in SARS-CoV-2 Spike: Evidence that D614G Increases Infectivity of the COVID-19 Virus. Cell. 2020 Aug 20;182(4):812–827.e19. PMCID: PMC7332439

![](https://els-jbs-prod-cdn.jbs.elsevierhealth.com/cms/attachment/3cacf135-842d-4065-b943-ffa6ffc806e8/gr6.jpg)

G614D may have selective advantage based on its wide spread. It is associated with higher viral load (see also http://dx.doi.org/10.1016/j.cell.2020.06.040) in patients.

Tags: D614G

------


http://dx.doi.org/10.1038/s41598-020-70812-6

Islam MR, Hoque MN, Rahman MS, Alam ASMRU, Akther M, Puspo JA, Akter S, Sultana M, Crandall KA, Hossain MA. Genome-wide analysis of SARS-CoV-2 virus strains circulating worldwide implicates heterogeneity. Sci Rep. 2020 Aug 19;10(1):14004. PMCID: PMC7438523

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41598-020-70812-6/MediaObjects/41598_2020_70812_Fig1_HTML.png)

The manuscript used GISAID data from March 30, 2020 to discover all possible sequence variants including SNP and indels. To be comprehensive, this likely needs to be redone for the current, much bigger, GISAID release.

Tags: Genomic diversity, GISAID

-------

http://dx.doi.org/10.7717/peerj.9703

Eskier D, Suner A, Karakülah G, Oktay Y. Mutation density changes in SARS-CoV-2 are related to the pandemic stage but to a lesser extent in the dominant strain with mutations in spike and RdRp. PeerJ. 2020 Aug 19;8:e9703. PMCID: PMC7443079

Analyzed GISAID (May 24, 2020) to find differences across complete SARS-CoV2 genomes. 


Tags: Genomic diversity, GISAID, C14408T (RdRp), A23403G (S)

--------

http://dx.doi.org/10.1128/JVI.00790-20

Liu Z, Zheng H, Lin H, Li M, Yuan R, Peng J, Xiong Q, Sun J, Li B, Wu J, Yi L, Peng X, Zhang H, Zhang W, Hulswit RJG, Loman N, Rambaut A, Ke C, Bowden TA, Pybus OG, Lu J. Identification of Common Deletions in the Spike Protein of Severe Acute Respiratory Syndrome Coronavirus 2. J Virol [Internet]. 2020 Aug 17;94(17). Available from: http://dx.doi.org/10.1128/JVI.00790-20 PMCID: PMC7431800

Authors performed passage of SARS-CoV2 in Vero E6 and Vero cells. Sequence analysis was performed using (1) ARTIC v1 followed by miSeq and ONT sequencing, (2) DRS on ONT, and (3) Sanger sequencing using the nCoV-2019_78_LEFT and nCoV-2019_78_RIGTH primers. Two deletion variants with S were investigated: the deletion of QTQTN was detected in 3 of 68 clinical samples and 12 of 24 in vitro-isolated viruses, while the deletion of
NSPRRAR was identified in 3 in vitro-isolated viruses.

![](https://jvi.asm.org/content/jvi/94/17/e00790-20/F1.large.jpg)

Tags: indels, S, ARTIC v1, ONT, miSeq, Sanger, DRS

------
